packages/nimble-tokens/dist/fonts/css/fonts.css

```
@@ -6,6 +6,13 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: "Source Sans Pro-fallback";
+  size-adjust: 93.75%;
```

I think this font should align for the same usages as the one it is being swapped with:

```suggestion
  font-style: normal;
  font-weight: 400;
  size-adjust: 93.75%;
```

If that makes sense please do that with the other instances

---

I understood your comment to mean I should explicitly put `font-style: normal` and `font-weight: 400` on the fallback fonts that did not specify those descriptors. I've done that now.